.github/workflows/building-with-pegof.yml

```diff
@@ -43,8 +43,8 @@ jobs:
       - run: make -j2
 
       - run: ./ctags --list-features | grep pegof
+      - run: ./ctags --list-languages=_packcc | tr '\n' ','
```

Comment:
The prefix "Targets:" is needed to help the developers (us) remember the meaning of the message.
How about adding the following line before the line you added?
```
      - run: print "Target: "
```

Reply:
But I wonder why the original caused a syntax error.

Reply:
because `Target: ` ':' follow a empty char. 

```
 - run: print "Target: "
```

This line has two map, one is `run:`, one is `Target:`. This is not allowed in YAML.

Reply:
Oh, I see. Today I hate computers even more!